webapp/src/components/Controls/SwitchToggler.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { FormGroup, FormControlLabel, Switch } from "@mui/material";
+
+type LabelPlacement = "top" | "bottom" | "start" | "end";
+
+type Props = {
+  label: string;
+  labelPlacement: LabelPlacement;
+  enable?: true;
+  onChange: (toggleValue: boolean) => void;
+};
+
+const SwitchToggler: React.FC<Props> = ({
+  label,
+  labelPlacement,
+  enable,
+  onChange,
+}) => {
+  const handleSwitchChange = (checked: boolean) => {
+    onChange(checked);
+  };
+  return (
+    <FormGroup aria-label="position" row>
+      <FormControlLabel
+        control={
+          <Switch
+            checked={enable ?? false}
+            onChange={(_, checked) => handleSwitchChange(checked)}
```

Comment:
Is there something missing or you can call `onChange` directly?
```suggestion
            onChange={(_, checked) => onChange(checked)}
```

Reply:
can directly pass to onchange. I did not think about it. thats a good approach